### DISCUSSION TITLE
Is it possible to move a running app to a different audio output?

### DISCUSSION THREAD

**Discussion by sla-te** [File-New-Project/EarTrumpet#842] [Q&A]
CheVolume used to have this possibility, sadly their website is offline now.

**Comment by Geeyun-JY3** [comment]
https://github.com/File-New-Project/EarTrumpet#features
> - Move apps between playback devices

I think this is what you want.

Expand the panel > right click the running app you want to move > click the swap button > choose a device

**Reply by sla-te** [reply]
aaaaah, omg thanks I didnt see that existed

**Reply by valipod** [reply]
You should mark this as answered, but neverthebless the functionality is broken in dev windows 10 builds, see https://github.com/File-New-Project/EarTrumpet/issues/835
